src/main/java/core/basesyntax/service/ShopServiceImpl.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service;
+
+import core.basesyntax.operation.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.storage.Storage;
+import core.basesyntax.model.FruitTransaction;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class ShopServiceImpl implements ShopService {
+    private final OperationStrategy operationStrategy;
+    private final Map<String, Integer> storage = Storage.getFruitStorage();
```

Comment:
According to the checklist, services should not interact directly with the storage. The storage interaction should be encapsulated within the operation handlers.